contracts/marketplace/Cargo.toml

```diff
@@ -11,6 +11,9 @@ scale-info = { version = "2.3", default-features = false, features = ["derive"],
 openbrush = { tag = "3.0.0", git = "https://github.com/727-Ventures/openbrush-contracts", default-features = false, features = ["ownable", "psp34", "reentrancy_guard"] }
 pallet_marketplace = { path = "../../logics", default-features = false }
 
+[dev-dependencies]
+ink_e2e = "4.0.1"
```

Comment:
Since you are updating this contract, please use the latest ink. I think it is 4.2